src/lib/y2storage/blk_device.rb

```diff
@@ -82,6 +82,36 @@ def blk_filesystem
       encrypted? ? encryption.direct_blk_filesystem : direct_blk_filesystem
     end
 
+    alias_method :filesystem, :blk_filesystem
+
+    # Label of the filesystem, if any
+    # @return [String, nil]
+    def filesystem_label
+      return nil unless blk_filesystem
+      blk_filesystem.label
+    end
+
+    # UUID of the filesystem, if any
+    # @return [String, nil]
+    def filesystem_uuid
+      return nil unless blk_filesystem
+      blk_filesystem.uuid
+    end
+
+    # Type of the filesystem, if any
+    # @return [Filesystems::Type, nil]
+    def filesystem_type
+      return nil unless blk_filesystem
+      blk_filesystem.type
+    end
+
+    # Mount point of the filesystem, if any
+    # @return [String, nil]
+    def filesystem_mountpoint
+      return nil unless blk_filesystem
+      blk_filesystem.mountpoint
+    end
```

Comment:
To be honest, just because we use the shorcuts in the tests to do things like

```ruby
expect(proposal.devices.partitions).to contain_exactly(
  an_object_having_attributes(id: Y2Storage::PartitionId::BIOS_BOOT),
  an_object_having_attributes(filesystem_mountpoint: "/"),
  an_object_having_attributes(filesystem_mountpoint: "/home"),
  an_object_having_attributes(filesystem_mountpoint: "swap")
)
```
or
```ruby
expect(sda3).to have_attributes(
          filesystem_mountpoint: "swap",
          filesystem_uuid:       "33333333-3333-3333-3333-33333333",
          filesystem_label:      "swap3",
          size:                  1.GiB - 1.MiB
)
```

Which I admit is a rather arbitrary criteria.